starboard/shared/starboard/player/player_worker.h

```diff
@@ -63,6 +63,8 @@ class PlayerWorker {
   // All functions of this class will be called from the JobQueue thread.
   class Handler {
    public:
+    struct HandlerResult;
```

Comment:
Need to forward declare HandlerResult to avoid a compile error on line 78, but it's awkward as HandlerResult is defined right after. I don't see anything in the style guide that covers this case of forward declaration - open to suggestions on better ways to handle it.